app/views/handlers/confirmation_email.py

```diff
@@ -92,21 +94,19 @@ def is_enabled(schema: QuestionnaireSchema):
 
 
 @dataclass
-class ConfirmationEmailFulfilmentRequest(FulfilmentRequest):
+class ConfirmationEmailTask:
     email_address: str
     session_data: SessionData
     schema: QuestionnaireSchema
 
-    def _payload(self) -> Mapping:
-        return {
-            "fulfilmentRequest": {
+    @property
+    def payload(self):
+        return json.dumps(
+            {
```

Comment:
We made a decision a while ago to use the `fulfilmentRequest` wrapper to be consistent with how we queue external requests from runner. I can't see why we are removing that. In doing this, we have also dropped the extra metadata (transaction id).